src/Rulesets/Address/CreateAddressRuleset.php

```diff
@@ -39,6 +41,10 @@ public function rules(): array
                 'nullable',
                 app(ModelExists::class, ['model' => Language::class]),
             ],
+            'advertising_state' => [
+                'string',
```

Comment:
nullable?

Reply:
would keep it not nullable as the column isnt nullable